src/scripts/OutSystems/OSUI/Utils/LayoutPrivateOnPostMessage.ts

```diff
@@ -0,0 +1,104 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OutSystems.OSUI.Utils.LayoutPrivate {
```

Comment:
Why is this in the LayoutPrivate?

Reply:
Since this affects the layout and we also have other methods that are adjusting the document and body and placed in the LayoutPrivate I believe this is the correct place to add it. Do you have any other suggestions?

Reply:
As you talked with O11 team, leave it as it is then.